kalite/distributed/management/commands/compileymltojson.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import print_function
+
+import json
+import os
+import yaml
+
+from django.core.management.base import NoArgsCommand
+
+BLACKLISTED_YAML_FILES = [
+    ".travis.yml",
+    "circle.yml"
+]
```

Comment:
I agree on this approach, I was thinking the same... like let's not have a management command where we need to add new yml files every time we add one.